docs/memlim.md

```diff
@@ -0,0 +1,43 @@
+# Memory Limiting
+
+Each of the individual services within the sidecar have a default
+memory limit. The memory limit is a maximum number of bytes that a service is 
```

Comment:
```suggestion
memory limit. The memory limit is a maximum number of MB that a service is 
```

Reply:
I wouldn't change this - it is explaining what the memory limit is, not the specifics of the environment variable.